pkg/controller/complianceremediation/complianceremediation_controller_test.go

```diff
@@ -37,30 +40,35 @@ func getMockedRemediation(name string, labels map[string]string, applied bool, s
 		},
 	}
 
+	mc := &mcfgv1.MachineConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "MachineConfig",
+			APIVersion: mcfgapi.GroupName + "/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{},
+		Spec: mcfgv1.MachineConfigSpec{
+			Config: igntypes.Config{
+				Ignition: igntypes.Ignition{
+					Version: igntypes.MaxVersion.String(),
+				},
+				Storage: igntypes.Storage{
+					Files: files,
+				},
+			},
+		},
+	}
+	unstructuredobj, _ := runtime.DefaultUnstructuredConverter.ToUnstructured(mc)
```

Comment:
Should this test go from a raw definition to unstructured instead (ie, rawObjectToUnstructured())? It might not make much difference but just closer to what would happen in practice.

Reply:
I actually didn't know how to define the MachineConfig as a raw object, hence why I converted it :smile: tips are welcome.